compose/cli/main.py

```diff
@@ -472,6 +472,7 @@ def run(self, project, options):
                                   to the host.
             -T                    Disable pseudo-tty allocation. By default `docker-compose run`
                                   allocates a TTY.
+            -w, --workdir=""      Override the workdir
```

Comment:
I just noticed, this should match what `docker run --help` says "Working directory inside the container"